cves/CVE-2018-13302.yml

```diff
@@ -342,7 +358,10 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The biggest mistake in this vulnerability was the assumption that only one frame
+    type was able to have this out of array vulnerability. They should have done
+    tests on the other types of frame vulnerabilities before deciding to only check one.
```

Comment:
A bit confusing here. Did they intentionally decide to only check one frame? or did they happen to just miss the other frame types?

Reply:
From what I saw, they intentionally checked one frame. They knew of other frame types but only figured that the specific frame type would be vulnerable to this problem. Will clarify!